src/types.ts

```diff
@@ -257,3 +257,9 @@ export const DailyStatisticsResult = t.type({
 
 export type DailyStatistics = t.TypeOf<typeof DailyStatistics>;
 export type DailyStatisticsResult = t.TypeOf<typeof DailyStatisticsResult>;
+
+const EmailRecord = t.type({
+  emailAddress: t.string,
+});
+
+export type EmailRecord = t.TypeOf<typeof EmailRecord>;
```

Comment:
Eh i think there is a misunderstanding, the task was to add "EMAIL" as a validation type to features